computer-vision/config/iphone.yaml

```diff
@@ -0,0 +1,23 @@
+%YAML:1.0
```

Comment:
So works on an iPhone for now?

Reply:
we can use pictures taken by an iPhone but it does not run on the phone.

Reply:
It was the best camera available for testing